maps/graph_edges/glitched_logic/kpa_basement_skip.py

```diff
@@ -0,0 +1,24 @@
+"""
+This file represents edges of the world graph that have to be added
+for Glitched Logic: Basement Skip
+"""
+edges_kpa_add_basement_skip_parakarry = [
+    #? Front Door Exterior Lava Door Right -> Front Door Exterior Hangar Door Bottom Left (ultra boots clip and parakarry)
+    {"from": {"map": "KPA_62",  "id": 2}, "to": {"map": "KPA_62",  "id": 0}, "reqs": [["Lakilester","GF_KPA16_ShutOffLava"],["Parakarry"],["UltraBoots"]], "pseudoitems": ["RF_Ch8_FirstGuardDoor"], "mapchange": False}, 
+    #? Guard Door 1 Door Left -> Guard Door 1 Fall In Trap Door
+    {"from": {"map": "KPA_81",  "id": 0}, "to": {"map": "KPA_81",  "id": 1}, "reqs": [["Bombette"], ["Lakilester"], ["Parakarry"], ["UltraBoots"]], "mapchange": False}, 
+]
+
+edges_kpa_add_basement_skip_lakilester = [
+    #? Front Door Exterior Lava Door Right -> Front Door Exterior Hangar Door Bottom Left (laki teleport)
+    {"from": {"map": "KPA_62",  "id": 2}, "to": {"map": "KPA_62",  "id": 0}, "reqs": [["Lakilester"],["can_climb_steps"]], "pseudoitems": ["RF_Ch8_FirstGuardDoor"], "mapchange": False}, 
```

Comment:
Not knowing what the laki teleport method looks like: Is `GF_KPA16_ShutOffLava` missing intentionally here?

Reply:
Turning off the lava wouldn't be required for logic I would think, it's a laki teleport through the wall that parakarry brings you on top of when you do the ultra boots clip on the block.

Reply:
Ah, I see. Yes, since Laki is required for the trick itself, being able to shut off the lava doesn't matter.